benchmark/main.cpp

```diff
@@ -154,7 +154,7 @@ do {                                                                        \
     mbedtls_printf(HEADER_FORMAT, TITLE);                                   \
     fflush(stdout);                                                         \
                                                                             \
-    for (i = 1, alarmed = 0, t.attach(alarm, 1.0); !alarmed; i++) {         \
+    for (i = 1, alarmed = 0, t.attach_s(alarm, 1); !alarmed; i++) {         \
```

Comment:
This depends on https://github.com/ARMmbed/mbed-os/pull/11236 being released

Reply:
That's the whole point of the PR and it says so in the description.